src/ifElse.test.js

```diff
@@ -1,11 +1,37 @@
 'use strict';
 
 describe('ifElse', () => {
-  // const { ifElse } = require('./ifElse');
+  const { ifElse } = require('./ifElse');
 
-  it('should ', () => {
+  it(`should call a 'first' function if 'condition' return is true`, () => {
+    const condition = jest.fn(() => true);
+    const first = jest.fn();
+    const second = jest.fn();
 
+    ifElse(condition, first, second);
+
+    expect(first).toHaveBeenCalledTimes(1);
+    expect(second).toHaveBeenCalledTimes(0);
+  });
+
+  it(`should call a 'second' function if 'condition' return is false`, () => {
+    const condition = jest.fn(() => false);
+    const first = jest.fn();
+    const second = jest.fn();
+
+    ifElse(condition, first, second);
+
+    expect(second).toHaveBeenCalledTimes(1);
+    expect(first).toHaveBeenCalledTimes(0);
   });
 
-  // write tests here
+  it(`should call a 'condition' function without arguments`, () => {
+    const condition = jest.fn(() => {});
+    const first = jest.fn();
+    const second = jest.fn();
+
+    ifElse(condition, first, second);
+
+    expect(condition).toHaveBeenCalledTimes(1);
```

Comment:
You checked that the condition function was called once, but you didn't verify that it was called without arguments. To ensure that the condition was called without arguments, add this check:
expect(condition).toHaveBeenCalledWith();